.github/workflows/ci.yml

```diff
@@ -2,6 +2,17 @@ name: Continuous integration
 
 on:
   pull_request:
+    paths-ignore:
+      - '.cookiecutter/*'
+      - '.editorconfig'
+      - '.gitignore'
+      - '.mailmap'
+      - '.pyup.yml'
```

Comment:
The `.mailmap` file hasn't been updated in 6 years. Although it is still used if you view historical commits, we could probably get away with removing it. What about `.pyup.yml` and `.editorconfig`?

Reply:
I think we could remove all three of those

Reply:
I've removed these files from the repo and from the `paths-ignore`: https://github.com/hypothesis/h/pull/7764, https://github.com/hypothesis/h/pull/7765, https://github.com/hypothesis/h/pull/7766, https://github.com/hypothesis/h/pull/7755/commits/f7344b40dcc0553735504f03c2fa192ed859f576